astrobee/behaviors/inspection/src/inspection.cc

```diff
@@ -177,7 +177,7 @@ double Inspection::GetDistanceToTarget() {
                                                               depth_cam_, target_size_x_, target_size_y_);
     }
   }
-  return -1;
+  return 1.0;
```

Comment:
I feel like you probably want to return something more obviously wrong?

Reply:
I actually reverted that bit after talking to Katie

Reply:
the 1.0m was the default focus distance, (if we're not adjusting the focus it should be 1m away), but that's a very different value from what we currently have as a default, so will fix this later, as it is only needed for ISAAC16